jp/components/grid_search.md

```diff
@@ -277,34 +277,34 @@ On the right in our input group, let's create three separate containers with the
 </igx-suffix>
 ```
 
-### API Summary
+### API まとめ
 
-In this article we implemented our own bar for the grid with some additional functionality when it comes to navigating between the search results. We also used some additional Ignite UI for Angular components like icons and inputs. The search API is listed below.
+検索結果のナビゲーションは、グリッドにカスタムバー機能を追加して実装しました。アイコンや入力などその他の Ignite UI for Angular コンポーネントも使用しています。以下は検索 API です。
 
-#### Methods
-The following methods are available on the **IgxGridComponent**:
-| Name | Type | Parameters |Description |
+#### メソッド
+以下のメソッドは **IgxGridComponent** で使用できます。
+| 名前 | 型 | パラメーター |説明 |
 | :--- | :--- | :--- | :--- |
-| `findNext` | number | The string to search and, optionally, if the search should be case sensitive (defaults to false). | Finds the next occurrence of a given string in the grid and scrolls to the cell if it isn't visible. Returns how many times the grid contains the string. |
-| `findPrev` | number | The string to search and, optionally, if the search should be case sensitive (defaults to false). | Finds the previous occurrence of a given string in the grid and scrolls to the cell if it isn't visible. Returns how many times the grid contains the string. |
-| `clearSearch` | void | N/A | Removes all the highlights in the grid. |
-| `refreshSearch` | number | N/A | Reapplies the existing search. Returns how many times the grid contains the last search. |
+| `findNext` | number | 検索文字列と検索で大文字と小文字の区別をするかどうか (デフォルトは false)。 | グリッドで文字列の次の出現を検索します。表示されていない場合はセルへスクロールします。グリッドに文字列が何回含まれるかを返します。|
+| `findPrev` | number | 検索文字列と検索で大文字と小文字の区別をするかどうか (デフォルトは false)。 | グリッドで文字列の前の出現を検索します。表示されていない場合はセルへスクロールします。 グリッドに文字列が何回含まれるかを返します。|
+| `clearSearch` | void | N/A | グリッドのすべての強調表示を削除します。 |
+| `refreshSearch` | number | N/A | 既存の検索を再適用します。グリッドに文字列が何回含まれるかを返します。 |
 
 <div class="divider"></div>
 
-The following methods are available on the **IgxGridCellComponent**:
-| Name | Type | Parameters |Description |
+以下のメソッドは **IgxGridCellComponent** で使用できます。
+| 名前 | 型 | パラメーター |説明 |
 | :--- | :--- | :--- | :--- |
-| `highlightText` | number | The string to search and, optionally, if the search should be case sensitive (defaults to false). | Highlights all occurrences of a string in a given cell. Return how many times the searched string is contained in the cell. |
+| `highlightText` | number | 検索文字列と検索で大文字と小文字の区別をするかどうか (デフォルトは false)。 | セルで文字列のすべての出現を強調表示します。検索文字列がセルに何回含まれるかを返します。 |
 | `clearHighlight` | void | N/A | Removes all the highlights in the cell. |
 
 <div class="divider"></div>
 
-#### Properties
-The following properties are available on the **IgxGridComponent**:
-| Name | Type | Description |
+#### プロパティ
+以下のプロパティは **IgxGridComponent** で使用できます。
+| 名前 | 型 | 説明 |
 | :--- | :--- | :--- |
-| `lastSearchInfo` | ISearchInfo | Contains information about the last performed search. |
+| `lastSearchInfo` | ISearchInfo | 最後に実行された検索についての情報を含みます。 |
 
 
 <div class="divider"></div>
```

Comment:
There is a section below here that, for some reason, I cannot add comments to.  This "Additional Resources" section needs to be localized!

Reply:
Translated.